lib/index.ts

```diff
@@ -110,6 +113,9 @@ const DEVICE_UNPLUG_TIMEOUT = 5000;
 // Delay (in ms) to wait when epRead throws an error
 const READ_ERROR_DELAY = 100;
 
+// Transfer delay
+const TRANSFER_DELAY_MS = 1000;
```

Comment:
This code works well with no delay added.
This change makes `node-raspberrypi-usbboot` 33 seconds slower for each device.